packages/core/src/middleware/koa-proxy-guard.ts

```diff
@@ -24,7 +24,10 @@ export default function koaSpaSessionGuard<
     }
 
     // Session guard
-    if (guardedPath.some((path) => requestPath.startsWith(path))) {
+    if (
+      guardedPath.some((path) => requestPath.startsWith(path)) &&
+      !ctx.request.URL.searchParams.get('preview')
+    ) {
```

Comment:
nit: Might be too much for a if condition. Better define a more meaningful const before going into the condition check

Reply:
@demonzoo updated please check thx.

Reply:
Looks good, thanks